SchoolProject/src/main/java/menu/TeacherSubMenu.java

```diff
@@ -0,0 +1,64 @@
+package menu;
+
+import java.util.Scanner;
+
+import commands.Invoker;
+import commands.TeacherReciver;
+import commands.teacherCommands.ListStudentsCommandImp;
+import commands.teacherCommands.SelectStudentCommandImp;
+import commands.teacherCommands.SetGradeStudentImp;
+import helpers.Grade;
+
+public class TeacherSubMenu {
```

Comment:
If we will have another menu in the sub menu, Will you create a new class to support that? I think you can have a Menu class only... and create some classes to implement it. TeacherMenu and StudentMenu have showMenu method() so, Menu could be a interface or abstract class.